projects/js-toolkit/packages/js-api/data-set/index.ts

```diff
@@ -19,26 +21,31 @@ export interface FDSCellRenderer {
 	(args: FDSCellRendererArgs): HTMLElement;
```

Comment:
I was tempted to add a `buildHTMLElement` symbol to `FDSCellRenderer` for simmetry, but refrained from doing so because:

1. It would be a breaking change and, even though we are still in experiment phase, I prefer not to do it.
2. It makes the implementation of a cell renderer a bit more cumbersome, because instead of exporting a default function, the user would need to export an object containing a reference to the function, which seems a bit convoluted.
3. If in the future we need to export more things for a cell renderer (as is the case with the two extra methods in the FDS filter) we can always mimic the structure of `FDSFilter` and add a runtime check in liferay-portal so that if the CX exports a bare function we wrap it into an object with a `buildHTMLElement` property. That way we don't break anything while still evolving the API.

Reply:
Done -> https://github.com/liferay/liferay-frontend-projects/pull/1172/commits/3e2f0863498737fbff31010546298351098c7e63